assets/scss/breadcrumb.scss

```diff
@@ -1,8 +1,15 @@
 .breadcrumb {
-  background-color: transparent;
+  background-color: $white;
```

Comment:
This makes the breadcrumbs on every page white, is that indeed correct?

Reply:
checked with design team, they confirmed this should affect all pages